src/fhir/1_Questionnaire-C-IDAS.json

```
@@ -637,7 +637,7 @@
       "type": "choice",
       "required": false,
       "linkId": "cs-idas-9",
-      "text": "Weight loss",
+      "text": "In the last week have you observed Weight loss in your care recipient?",
```

Not sure if "Weight" here shouldn't be capitalized here.
```suggestion
      "text": "In the last week have you observed weight loss in your care recipient?",
```